### DISCUSSION TITLE
[BUG] Broken comment UI in locked issue (website)

### DISCUSSION THREAD

**Discussion by jodydonetti** [community/community#75195] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hi, when viewing a locked issue it appears the (recently changed) comment form UI is broken.
For example in [this issue](https://github.com/dotnet/runtime/issues/31542):

![image](https://github.com/community/community/assets/1010086/56612e1c-b7eb-4d06-8338-91e2fe0b3bdb)

you can see that the 2 tabs ("Write" and "Preview") appear, but they are not working, not clickable, and most definitely should not be there.

I would've expected something like this:

![image](https://github.com/community/community/assets/1010086/82127d82-18ef-46c0-986f-e1e77b3c44b2)

without the useless partial, non interacting tab.

Thoughts?

**Comment by Harrisbaig7** [comment]
Hi @jodydonetti ,

Thank you for reporting this issue and providing the screenshots. I can see that the comment form UI is broken when viewing a locked issue.

This indeed seems like unexpected behavior, and I appreciate you bringing it to our attention. I'll investigate this further to understand the root cause and work on a fix.

In the meantime, if you have any additional information or steps to reproduce the issue, please share them. Your input is valuable in helping us address this problem more efficiently.

I'll keep you updated on the progress, and thank you for your patience.

Best regards,
Muhammad Harris Baig

**Comment by jodydonetti** [comment]
Hi, any news?
